dev/snippets/bin/snippets

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Locate the pub-cache directory.
+PUB_CACHE=${PUB_CACHE:-$HOME/.pub-cache}
+
+"${PUB_CACHE}/bin/snippets" --output-directory="$FLUTTER_ROOT/dev/docs/doc/snippets" "$@"
```

Comment:
Since we "pub global activate"ed it, why not invoke it via `pub global run` or whatever the pub command is?

Reply:
Whoa.  Good idea. I'll check and see if that works.